### PR TITLE
OADP-339: Introduce stable-1.0 channel

### PR DESCRIPTION
**Pull request by rayfordj**

### CODE REVIEW

**Comment by rayfordj** [comment]
It looks like a lot changes caused by line-wrapping & intending performed by operator-sdk were introduced.

**Comment by rayfordj** [comment]
@kaovilai , please provide input on `make bundle`'s usage of `yq` because it was failing for me.  I'm not confident in what its expected to be doing.

```
yq: error: argument files: can't open 'del(.spec.install.spec.deployments.1)': [Errno 2] No such file or directory: 'del(.spec.install.spec.deployments.1)'
```

Thanks!

**Comment by kaovilai** [comment]
yq usage is a trick around operator-sdk to add velero service account into CSV
https://coreos.slack.com/archives/C0144ECKUJ0/p1638394991106700?thread_ts=1638283821.071800&cid=C0144ECKUJ0

**Comment by rayfordj** [comment]
> It looks like a lot changes caused by line-wrapping & indenting performed by operator-sdk were introduced.

After sorting out the `yq` failing, the changes are now more representative of what's expected.

**Comment by kaovilai** [comment]
On that note.. I think operator-sdk may have fixed the issue.
https://github.com/operator-framework/operator-sdk/issues/4751

**Comment by rayfordj** [comment]
@shawn-hurley , particularly interested in your review/ack to ensure we're capturing things as expected.  Thanks!

cc: @weshayutin

File: build/Dockerfile.bundle [hunk 1]
```diff
@@ -5,9 +5,9 @@ LABEL operators.operatorframework.io.bundle.mediatype.v1=registry+v1
 LABEL operators.operatorframework.io.bundle.manifests.v1=manifests/
 LABEL operators.operatorframework.io.bundle.metadata.v1=metadata/
 LABEL operators.operatorframework.io.bundle.package.v1=oadp-operator
-LABEL operators.operatorframework.io.bundle.channels.v1=stable
-LABEL operators.operatorframework.io.bundle.channel.default.v1=stable
-LABEL operators.operatorframework.io.metrics.builder=operator-sdk-v1.14.0
+LABEL operators.operatorframework.io.bundle.channels.v1=stable,stable-1.0
+LABEL operators.operatorframework.io.bundle.channel.default.v1=stable-1.0
+LABEL operators.operatorframework.io.metrics.builder=operator-sdk-v1.18.0+git
```
Comment (dymurray) [inline build/Dockerfile.bundle:10]: Unfortunately I don't know of a way to fix the `git` suffix, this happens with everyones installs on linux. So it's weird but fine by me. If you wanted ot manually remove it that works too.

**Review by dymurray — APPROVED** [review]

**Pull request merged**